roles/wazuh/ansible-wazuh-manager/tasks/main.yml

```diff
@@ -7,6 +7,34 @@
       - tar
     state: present
 
+- name: Check if NodeJS service Exists
+  stat:
+    path: /usr/bin/node
+  register: node_service_status
+
+- name: Installing NodeJS repository script
+  become: true
+  get_url:
+    url: "https://{{ repo_dic[ansible_os_family|lower] }}.nodesource.com/setup_8.x"
+    dest: /etc/nodejs.sh
+    mode: '0775'
+  changed_when: false
+  when: not node_service_status.stat.exists
```

Comment:
Englobe all tasks that need the `when: not node_service_status.stat.exists` under a `block` in order to apply the conditional to all of them and avoid code duplication

Reply:
Implemented in 5fd5ddc61f9befa864c7e37d92698ad48fffa50c